.credo.exs

```diff
@@ -79,7 +79,7 @@
         {Credo.Check.Refactor.ABCSize, max_size: 50},
         # {Credo.Check.Refactor.CaseTrivialMatches}, # deprecated in 0.4.0
         {Credo.Check.Refactor.CondStatements},
-        {Credo.Check.Refactor.FunctionArity, max_arity: 6},
+        {Credo.Check.Refactor.FunctionArity, max_arity: 9},
```

Comment:
may not change this

Reply:
Still need to keep this for internal function. But I have changed the public interface to make it takes less parameters.